FirebaseFunctionsSwift/Sources/FunctionsError.swift

```diff
@@ -140,7 +146,7 @@ extension FunctionsErrorCode {
     switch name {
```

Comment:
This and the next function could go away if we switched the raw value of the enum, but that's a breaking source change unfortunately (changing the value of the enums, and the underlying type). Likely wouldn't affect anyone but shouldn't do it now.